codebuild_specs/pr_workflow.yml

```diff
@@ -22,6 +22,10 @@ batch:
       buildspec: codebuild_specs/test.yml
       env:
         compute-type: BUILD_GENERAL1_LARGE
+    - identifier: lint
+      buildspec: codebuild_specs/lint.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
```

Comment:
can you add to the other workflows

Reply:
This one is needed only in PR builds see https://github.com/aws-amplify/amplify-cli/blob/392f0e2c342408b914aa1802d3520191ed70c9fd/.circleci/config.base.yml#L1064-L1068 .

Reply:
but it's included in `dev` build... so maybe it's worth adding.